app/ai/voice/agents/breeze_buddy/managers/lead_dispatcher.py

```
@@ -0,0 +1,333 @@
+            await self._try_enqueue(lead_id)
+        else:
+            # Schedule for later
+            score = next_attempt_at.timestamp()
+            try:
+                redis = await get_redis_service()
+                await redis.zadd(SCHEDULED_LEADS_KEY, {lead_id: score})
+                self._schedule_event.set()  # wake scheduler to recalculate sleep
+                logger.info(
+                    f"Lead {lead_id} scheduled for {next_attempt_at.isoformat()}"
+                )
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)